### PR TITLE
Photon Controller Driver: Fixed Go format issues and ssh issues with boot2docker.

### PR DESCRIPTION
**Pull request by faijaz**
Pull request includes:
1) Fixes for gofmt.
2) Fixes for ssh using boot2docker.

### CODE REVIEW

File: drivers/photoncontroller/photoncontroller.go [hunk 1]
```diff
@@ -367,32 +367,36 @@ func (d *Driver) Remove() error {
...
-	if (err != nil) {
-		return fmt.Errorf("Error starting VM: ", err)
+	if err != nil {
+		return fmt.Errorf("Error starting VM: %s", err.Error())
 	}
 
 	d.waitForVM()
 
+	if d.ISOPath == "" {
+		d.CopyPublicSSHKey()
```
Comment (AlainRoy) [inline drivers/photoncontroller/photoncontroller.go]: Can you explain why you need to copy the SSH key in Start()? You already did it in Create(), so I'm not sure why it's needed again.

File: drivers/photoncontroller/photoncontroller.go [hunk 2]
```diff
@@ -401,18 +405,22 @@ func (d *Driver) Stop() error {
...
-	if (err != nil) {
-		return fmt.Errorf("Error restarting VM: ", err)
+	if err != nil {
+		return fmt.Errorf("Error restarting VM: %s", err.Error())
 	}
 
 	d.waitForVM()
 
+	if d.ISOPath == "" {
+		d.CopyPublicSSHKey()
```
Comment (AlainRoy) [inline drivers/photoncontroller/photoncontroller.go]: Same question here.

File: drivers/photoncontroller/photoncontroller.go (hunk 1 above)
Comment (faijaz) [inline drivers/photoncontroller/photoncontroller.go]: Fixed it.

File: drivers/photoncontroller/photoncontroller.go (hunk 2 above)
Comment (faijaz) [inline drivers/photoncontroller/photoncontroller.go]: Fixed it.

**Comment by AlainRoy** [comment]
Looks good to me!

**Pull request merged**